config.go

```diff
@@ -82,7 +82,7 @@ type Config struct {
 
 // This constant sets the default endpoint to which client instances send
 // messages if none was explictly set.
-const DefaultEndpoint = "https://localhost:8080"
+const DefaultEndpoint = "https://api.dreamdata.cloud"
```

Comment:
yeah, this is an excellent point. We only want to override when it is `api.segment.com`

Reply:
It points to `localhost:8080` because @paddie changed it in 395fc3983887620bf8def43d3ac58f63f8690c69. I do believe that here is where we want to point to out prod system.

Reply:
@miquelpuigmena ahh, yes, the diff confused me. I was trying to figure out at what point the original repo would override `localhost:8080` when using the client and totally missed my original change.

Reply:
The original line is here @wizhi @miquelpuigmena https://github.com/segmentio/analytics-go/blob/v3.0/config.go#L85

the localhost is totally me trying to help Jess test locally against her version of the api